pkg/virt-controller/watch/network_test.go

```diff
@@ -220,6 +221,71 @@ var _ = Describe("Network interface hot{un}plug", func() {
 			),
 			!ordinal),
 	)
+
+	DescribeTable("interface with 'absent' state",
+		func(specIfaces []v1.Interface, statusIfaces []v1.VirtualMachineInstanceNetworkInterface,
+			expectedInterfaces []v1.Interface, expectedNetworks []v1.Network) {
+			vmi := api.NewMinimalVMI("test")
+
+			var testNetworks []v1.Network
+			for _, iface := range specIfaces {
+				testNetworks = append(testNetworks, v1.Network{Name: iface.Name})
+			}
+			vmi.Spec.Networks = testNetworks
+			vmi.Spec.Domain.Devices.Interfaces = specIfaces
+			vmi.Status.Interfaces = statusIfaces
+
+			vmiSpecCopy := vmi.Spec.DeepCopy()
+
+			clearDetachedInterfaces(vmiSpecCopy, statusIfaces)
+
+			Expect(vmiSpecCopy.Domain.Devices.Interfaces).To(Equal(expectedInterfaces))
+			Expect(vmiSpecCopy.Networks).To(Equal(expectedNetworks))
+		},
+		Entry("should remain in spec, given 2 non-absent interfaces, and associated status ifaces",
+			[]v1.Interface{
+				{Name: "blue"},
+				{Name: "red"},
+			},
```

Comment:
Can you use one-liner here and below? The current format bloats the test to be very big with no special need.